omcb.py

```diff
@@ -113,6 +117,24 @@ def apply_line_to_state(state, line, after=None, before=None):
 
     return (state, date, "continue")
 
+def apply_line_to_state_heatmap(state, line, after=None, before=None):
```

Comment:
I think this is 1-1 with the timelapse equivalent code? maybe not relevant if you agree with my comment below that we don't really need to manage state for heatmaps in the same way, but if we do need this code we can just rename our function for applying a line to the timelapse state and use it for heatmaps too.

Reply:
almost 1-1! I was running into type errors with cell so I changed it but didn't want to significantly change timelapse logic. It can likely just be combined with the other function and add an int() wrap around cell. Will look into exactly why I was getting type errors with int when I get home